addons/account/models/account_move.py

```diff
@@ -1942,6 +1943,28 @@ def _compute_tax_country_code(self):
         for record in self:
             record.tax_country_code = record.tax_country_id.code
 
+    @api.depends('ref', 'move_type', 'partner_id', 'invoice_date')
+    def _compute_duplicated_ref(self):
+        """ The computed field only appears in the move view form and thus is supposed
+        to only be computed for one move at a time which makes the search in loop "ok"
+        """
```

Comment:
This is not documentation, but a comment

Reply:
Indeed, dumb me!
Nice catch!